src/main/resources/META-INF/plugin.xml

```diff
@@ -80,24 +80,19 @@
   ]]>
   </change-notes>
 
-  <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
```

Comment:
You need to claim the minimum version as JetBrains validation will fail when plugin will be submitted

Reply:
ack!

Reply:
Kept in sync with tekton.